extensions/kill-process/package.json

```diff
@@ -31,7 +32,7 @@
       "type": "checkbox"
     },
     {
-      "description": "Determines whether the pid of the process should be searched for the query, or only the process name.",
+      "description": "Determine whether the process PID should be searched for the query or only the process name.",
```

Comment:
Not sure why, but "determines" sounds more natural to me, because the flag/checkmark determines something. "Determine" is an action.

Reply:
"Determine" is in the imperative form. It gives an instruction to the user. Do you want me to revert back this change?